torch/ao/quantization/fx/_model_report/detector.py

```diff
@@ -189,9 +192,10 @@ def _detect_per_channel_helper(self, model: nn.Module, per_channel_info: Dict):
                     else:
                         raise ValueError("Should be either observer or fake quant")
 
-                per_channel_info["per_channel_status"][fqn] = {
-                    "per_channel_supported": per_channel_supported,
-                    "per_channel_used": per_channel_used,
+                per_channel_info[fqn] = {
+                    self.PER_CHAN_SUPPORTED_KEY: per_channel_supported,
+                    self.PER_CHAN_USED_KEY: per_channel_used,
+                    self.BACKEND_KEY: self.backend_chosen
```

Comment:
nit: not sure if there's a more elegant way to do this since its more of a model level piece of info than a module level...

Reply:
i initially had it at a higher level since it was a model level piece and not module specific, but that was the issue in terms of consistent "levels" for features across all the different detectors, which is why i just decided to include it at this level. It makes the Visualizer much easier